discovery-aws-api/src/main/scala/akka/discovery/awsapi/ec2/Ec2TagBasedSimpleServiceDiscovery.scala

```diff
@@ -34,7 +36,10 @@ class Ec2TagBasedSimpleServiceDiscovery(system: ActorSystem) extends SimpleServi
 
   override def lookup(name: String, resolveTimeout: FiniteDuration): Future[Resolved] = {
 
-    val ec2Client = AmazonEC2ClientBuilder.defaultClient
+    // we have our own retry/backoff mechanism, so we don't need EC2Client's in addition
+    val clientConfiguration = new ClientConfiguration()
+    clientConfiguration.setRetryPolicy(PredefinedRetryPolicies.NO_RETRY_POLICY)
+    val ec2Client = AmazonEC2ClientBuilder.standard().withClientConfiguration(clientConfiguration).build()
```

Comment:
good thanks :) We don't have a backoff yet but yes, let's keep as is here as we want to have a backoff on the outside indeed